configuration/pih/petl/jobs/datawarnings/job.yml

```diff
@@ -0,0 +1,15 @@
+type: "sqlserver-bulk-import"
+configuration:
+  extract:
+    datasource: "mysql/openmrs.yml"
+    query:  "datawarnings/source.sql"
+
+  load:
+    datasource: "sqlserver/openmrs_extractions.yml"
+    table: "hivmigration_data_warnings"
+    schema: "datawarnings/target.sql"
+
+  conditional: "SELECT COUNT(*)=1 FROM information_schema.tables WHERE table_schema = 'openmrs' AND table_name = 'hivmigration_data_warnings'"
```

Comment:
If the 95% use case for the conditional is going to be to check the existence of a table, would be helpful to have a more convenient way to express that, though this is fine for now.  That could even just be creating a "table_exists" function that takes in the table name and returns a boolean, which would allow the conditional to be "select table_exists('hivmigration_data_warnings')"

Reply:
Makes sense, thanks @mseaton